apps/fluent-tester/src/FluentTester/theme/ThemePickers.tsx

```diff
@@ -1,26 +1,38 @@
 import * as React from 'react';
-import { View, Picker, StyleSheet } from 'react-native';
+import { View, StyleSheet, Platform } from 'react-native';
 import { Text } from '@fluentui-react-native/experimental-text';
 import { lightnessOptions, testerTheme } from './CustomThemes';
 import { themeChoices, ThemeNames } from './applyTheme';
 import { brandOptions, OfficeBrand } from './applyBrand';
+import { Theme, useTheme } from '@fluentui-react-native/framework';
+import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
+
+const isAndroid = Platform.OS == 'android';
+
+// To be replaced with single import when Windows support comes in.
+const Picker = isAndroid ? require('@react-native-picker/picker').Picker : require('react-native').Picker;
 
 export const themePickerStyles = StyleSheet.create({
   pickerRoot: {
     flexDirection: 'row',
   },
 
   picker: {
-    flexDirection: 'row',
+    flexDirection: isAndroid ? 'column' : 'row',
     alignItems: 'center',
     padding: 4,
   },
+});
 
-  dropdown: {
-    height: 30,
-    width: 120,
-    fontSize: 12,
-  },
+export const getThemedDropdownStyles = themedStyleSheet((t: Theme) => {
+  return {
+    dropdown: {
+      height: 30,
+      width: 120,
+      fontSize: 12,
+      color: t.colors.bodyText,
+    },
+  };
```

Comment:
The picker menu looked like it was light mode even in dark mode. Is there a `backgroundColor` type prop we can set here to help with that?

Reply:
There is no prop/style of backgroundColor in picker. The background color is set based on system (Android UI mode) theme. This has something to do with reactInstanceManager updating configuration on receiving notification from system. (Even auto switching of theme doesn't work when system theme is  switched). This issue is on react-native-test-app's end (I think or I am unaware of the right way to get this configuration working on the app). Talking with @arazabishov to get this working. 

Offline I have got it working by modifying the RNTA in `node_modules`.

https://user-images.githubusercontent.com/23552252/115561699-746c8400-a2d3-11eb-80ab-026e784a8e2f.mp4